adaway-linux.sh

```diff
@@ -62,54 +61,60 @@ while read src; do
         # - remove additional localhost entries possibly picked up from sources
         # - remove remaining comments
         # - split all entries with one tab
-        if type curl 2>/dev/null > /dev/null ; then
-          DOWNLOAD_CMD=$(curl --progress-bar -L "${src}")
+        if type curl 1>/dev/null 2>&1; then
+          DOWNLOAD_CMD=$(curl --progress-bar -L --connect-timeout 20 --retry 2 "${src}")
         else
-          DOWNLOAD_CMD=$(wget "${src}" -nv --show-progress -L -O -)
+          DOWNLOAD_CMD=$(wget "${src}" -nv --show-progress --read-timeout=20 --timeout=20 -t 2 -L -O -)
```

Comment:
Good idea. But we should add a variable `DL_TIMEOUT` and `DL_RETRIES` or something like that.